code/game/objects/structures/fluff.dm

```diff
@@ -1355,3 +1355,109 @@
 	stake.forceMove(drop_location())
 	stake = null
 	qdel(src)
+
+/obj/structure/fluff/statue/shrine/donation
+    parent_type = /obj/structure/fluff/statue	
```

Comment:
there's no reason to mess with the parent_type like this for this structure-- having to do this means something has gone horribly wrong. https://ref.harry.live/datum/var/parent_type 

why not just have it be shrine/donation/abyssorqueststatue (etc for other gods)??? then you could define the `icon` var to `icon = 'icons/roguetown/misc/64x96.dmi'` on only that parent and not duplicate it to the children, same for `max_integrity = null`.